src/pages/Home/Home.page.jsx

```diff
@@ -1,39 +1,42 @@
-import React, { useRef } from 'react';
-import { Link, useHistory } from 'react-router-dom';
+import React, { useCallback, useEffect, useState } from 'react';
+import styled from '@emotion/styled';
 
-import { useAuth } from '../../providers/Auth';
-import './Home.styles.css';
+import { $getVideos } from '../../api/videos';
 
-function HomePage() {
-  const history = useHistory();
-  const sectionRef = useRef(null);
-  const { authenticated, logout } = useAuth();
+import GridVideo from '../../components/GridVideo';
+import Container from '../../components/Common/Container';
 
-  function deAuthenticate(event) {
-    event.preventDefault();
-    logout();
-    history.push('/');
-  }
+const HomePageContainer = styled(Container)`
+  margin: auto;
+`;
+
+const HomePage = () => {
+  const [state, setState] = useState({
+    isLoading: true,
+    videos: [],
+  });
+
+  const fetchVideos = useCallback(async () => {
```

Comment:
`useCallback` is useless here unless we pass this function to child components
```
This is useful when passing callbacks to optimized child components
```

More about `useCallback`: https://reactjs.org/docs/hooks-reference.html#usecallback